src/api/lib/tasks/rollout.rake

```diff
@@ -1,47 +1,49 @@
 # rubocop:disable Rails/SkipsModelValidations
+
+# USAGE NOTES:
```

Comment:
I think we should point to the wiki page below instead of having these usage notes:

https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#steps-for-rolling-out

This will remove the documentation duplication. It prevents us from having outdated documentation here or in the wiki.

Reply:
I decided not to do so because the Wiki documentation is so changeable that, sooner or later, we'll end up with a broken link.

Reply:
I also thought about that when writing my comment. Let's see what others think. I'm fine if we leave it like this.

Reply:
I think that if we have a wiki, we have to maintain it. If it has broken links, we fix them. But if we add documentation in the code then we have two different sources, and then we have to maintain both (otherwise we'll have documentation saying different things, which IMO is worse that having no documentation).
So for me, +1 for having it on the wiki.

Reply:
I think as well that we should link to the Wiki, however we should rename this page. Flipper is implementation detail and shouldn't be in the title. The title should be something like "Features rollout" or "How to rollout new features"

Reply:
Alright, so it seems like we agree on having a link to the wiki.

Reply:
Comment updated pointing to the wiki. Thanks for giving your opinion. :bouquet: